src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -61,14 +62,18 @@ public PoliciesController(
     public async Task<PolicyResponseModel> Get(string orgId, int type)
```

Comment:
Lets take this opportunity to change the `orgId` type to `Guid` so its automatically parsed
```suggestion
    public async Task<PolicyResponseModel> Get(Guid orgId, int type)
```